pkg/openid4ci/clientconfig.go

```diff
@@ -66,3 +70,21 @@ func validateClientConfig(config *ClientConfig) error {
 
 	return nil
 }
+
+func setDefaultsAsNeeded(config *ClientConfig) {
```

Comment:
I feel setDefault is self explanatory. ie, without the suffix "AsNeeded".

Reply:
Done - changed to `setDefaults`